chart/openfaas-cloud/values.yaml

```diff
@@ -22,6 +22,7 @@ edgeAuth:
   writeDebug: false
   ## clientSecret is the value from your OAuth application
   clientSecret:
+  secureCookie: "false"
```

Comment:
Should we default to true?

Reply:
the default tls.enabled value is "False" we could use the same config value to work out if we should enable it rather than bringing in a new one?